lib/rules/no-double-assignments.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
```

Comment:
Please see other rules for examples of how to properly format your rule JS. You can also use the ESLint yeoman generator to automatically setup a skeleton file.